pyspectral/near_infrared_reflectance.py

```diff
@@ -108,6 +108,7 @@ def __init__(self, platform_name, instrument, band,
         self.lutfile = None
 
         platform_sensor = platform_name + '-' + instrument
+        tb2rad_dir = options.get('tb2rad_dir', tempfile.gettempdir())
```

Comment:
@adybbroe This tempdir creation as a default differs from the previous implementation in `utils.py` because now a tempdir is created for every instance of this class. In `main` this directory is only created once. Do you know if that is a problem for this Calculator object? I mean, is it a performance problem that the directory changes for every instance?